site/src/main/scala/org/typelevel/sbt/TypelevelSitePlugin.scala

```diff
@@ -128,15 +148,16 @@ object TypelevelSitePlugin extends AutoPlugin {
             .toAbsolutePath
             .relativize(((Laika / target).value / "site").toPath)
             .toString,
-          "publish_branch" -> "gh-pages"
+          "keep_files" -> tlSiteKeepFiles.value.toString
         ),
         name = Some("Publish site"),
         cond = {
-          val predicate = tlSitePublishBranch
-            .value // Either publish from branch or on tags, not both
```

Comment:
When I first wrote this, I was afraid about race conditions when publishing the same commit twice, for a branch and for a tag. But I think it's fine.